apps/portal/app/consts/general.ts

```diff
@@ -29,7 +29,8 @@ export const RELIC_CONTRACT_ADDRESS =
 
 export const DEFAULT_LIMIT = 10
 
-export const MIN_DEPOSIT = CURRENT_ENV === 'development' ? '0.00069' : '0.00042'
+export const MIN_DEPOSIT =
```

Comment:
This is the fallback that should only be utilized when we don't get it directly from the RPC via the `getVaultDetails` -- mainly just clarifying. 

Looks good!